tool/lib/test/unit/testcase.rb

```diff
@@ -198,22 +215,24 @@ def run runner
 
       RUN_TEST_TRACE = "#{__FILE__}:#{__LINE__+3}:in `run_test'".freeze
       def run_test(name)
-        progname, $0 = $0, "#{$0}: #{self.class}##{name}"
+        progname, $0 = $0, "#{$0}: #{self.class}##{name}" if main_ractor?
```

Comment:
TODO:
if we make main Ractor as a manager Ractor (and other Ractors run tests, a manager Ractor knows which tests are running), we can set a running tests on the main Ractor.